rust/opendp_tooling/src/codegen/r/mod.rs

```diff
@@ -43,6 +43,7 @@ const BLACKLIST: &'static [&'static str] = &[
     "user_domain",
     "_make_transformation",
     "_make_measurement",
+    "_new_pure_function",
```

Comment:
If this weren't here, I wouldn't notice that it was missing. How do we make sure this list is up to date?

Reply:
R will start to fail!